web/beacon-app/src/components/layout/Sidebar/Sidebar.tsx

```diff
@@ -40,7 +40,7 @@ const otherMenuItems: MenuItem[] = [
   {
     name: 'Support',
     icon: <SupportIcon />,
-    href: routes.supports,
+    href: routes.support,
```

Comment:
I think we'll need to change this to mailto:support@rotational.io for now. We'll also need to route any links to docs to here: https://ensign.rotational.dev/getting-started/